oggm/core/models/flowline.py

```diff
@@ -300,8 +300,14 @@ def __init__(self, flowlines, mb_model=None, y0=0., fs=None, fd=None):
         """
 
         self.mb = mb_model
-        self.fd = fd
+        # self.fd = fd
         self.fs = fs
+        # for current backwards compatibility I calculate fd from Aglen if fd is not given
+        # We should however streamline oggm to only use Aglen and get rid of fd
+        if fd is not None and Aglen is None:
+            Aglen = (N+2.) * fd / 2.
+            
+        self.Aglen = Aglen
```

Comment:
I have changed this here to calculate Aglen from fd only if fd exists but Aglen not, else we should use Aglen. This works for all the tests I ran, but some of them still return failed. The reason is that your original fd is set to quite a different value. 

In your TestIdealisedCases class you set ``self.fd = 1.9e-24`` and I set ``self.Aglen = 2.4e-24``. Converting the fd into the Aglen world you end up with your fd being equivalent to an Aglen of 4.75e-24, which is way softer ice than usually assumed. The perceived softer ice makes the benchmarks fail I guess.